_episodes/05-accelerating-lammps.md

```diff
@@ -570,83 +602,88 @@ before starting the production runs. This might save your lot of resource and ti
-> > 2. Fix Me.
+> >    very low. The system size is so small that considering GPU acceleration is practically
+> >    meaningless. In this case you are emplying far too many workers to complete a small job and
+> >    therefore it is not surprising that the scaling deteriorates with increasing MPI tasks. In
+> >    fact, you may try to use 1 GPU and a few MPI task to see if the performance increases in
+> >    this case.
+> > 2. (**FIXME**).
 > {: .solution}
 {: .challenge}
 
-> ## Exercise: Switch on dynamic load balancing
+#### **Load balancing: revisited**
+
+We have discussed load balancing in the previous episode due to the potential of underperformance
+in systems with an inhomogeneous distribution of particles. We discussed earlier that it can be
+done using the `split` keyword. Using this keyword a fixed fraction of particles is offloaded to
+the GPU while force calculation for the other particles occurs simultaneously on the CPU. When you
+set `split` = `1` you are offloading entire force computations to the GPUs (discussed in previous
+exercise). What fraction of particles would be offloaded to GPUs can be set explicitly by choosing
+a value ranging from `0` to `1`. When you set its value to `-1`, you are switching on dynamic load
+balancing. This means that LAMMPS selects the split factor dynamically.
+
+> ## Switch on dynamic load balancing
 >
-> We discussed earlier that it can be done using the `split` keyword. Using this keyword a
-> fixed fraction of particles is offloaded to the GPU while force calculation for the
-> other particles occurs simultaneously on the CPU. When you set `split 1` you are
-> offloading entire force computations to the GPUs (discussed in previous exercise). What
-> fraction of particles would be offloaded to GPUs can be set explicitly by choosing a
-> value ranging from `0` to `1`. When you set its value to  `-1`, you are switching on
-> dynamic load balancing. This means that LAMMPS selects the split factor dynamically.
 >
-> Let us repeat the entire exercise as described in the
-> [previous exercise](#exercise-offload-entire-neighbor-build-and-force-computation-to-gpus)
-> but this time we'll use dynamic load balancing, i.e. in the command-line we'll use:
-> ```
-> -sf gpu -pk gpu 4 neigh yes newton off split -1.0
-> ```
-> Plot the data for *normalized speed-up factor per node* versus GPU/MPI choices for each
-> of these configurations and discuss how it differs from the observations that you
-> derived from the
-> [previous exercise](#exercise-offload-entire-neighbor-build-and-force-computation-to-gpus).
+> Let us repeat the entire exercise as described in the previous exercise
+> but this time we'll use dynamic load balancing, changing the value of `split` to `-1.0`. Run your
+> setup again and compare the performances.
 >
 > > ## Solution
-> > (FIX ME) This needs to be done!
+> > 
+> > Using the same setup as previously, you can see how the plot has changed. **(FIX ME)** This needs to be done!
```

Comment:
@prithwish-ichec we need more info in here on the difference in plot... but this may not be needed because it is in load balancing section as discussed in last meeting? Could always get rid of it? @ocaisa, any thoughts?

Reply:
The plot says it all. Dynamic load balancing increasing the performance. If required, we can add a few extra lines later.